nekoyume/Assets/_Scripts/Game/Factory/ItemMoveAnimationFactory.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Nekoyume.EnumType;
+using Nekoyume.Game.Util;
+using Nekoyume.UI;
+using Nekoyume.UI.Module;
+using UnityEngine;
+
+namespace Nekoyume.Game.Factory
+{
+    public static class ItemMoveAnimationFactory
+    {
+        private static ObjectPool _objectPool;
+        private static bool _initialized;
+
+        public enum AnimationItemType
+        {
+            Crystal,
+            Ncg,
+        }
+
+        private const string CrystalAnimationPrefabName = "item_CrystalGetAnimation";
+
+        private static void Initialize()
+        {
+            if (!_initialized)
+            {
+                _initialized = true;
+                _objectPool = MainCanvas.instance.GetLayerRootTransform(WidgetType.Animation)
+                    .GetComponent<ObjectPool>();
+                _objectPool.Initialize();
+            }
+        }
```

Comment:
How about using `Lazy` statement?
https://docs.microsoft.com/ko-kr/dotnet/api/system.lazy-1?view=net-6.0

Reply:
Thanks to your sugesstion. I will check it and reflect if it can!